packages/ui/src/components/Dialog/Dialog.tsx

```diff
@@ -125,51 +119,58 @@ DialogClose.displayName = DialogPrimitive.Close.displayName;
 const DialogHeader = ({
   className,
   size,
```

Comment:
Maybe we can default this value to `lg` and simplify the code below a bit (I'll add a comment where I'm talking about)